### PR TITLE
Improve : Improved the design on landing page.

### PR DESCRIPTION
**Pull request by AMANVISHWAKARMA27**
The button in the landing page has been made equal to the width of other elements. The placeholders have been aligned correctly.
Font-bold -> font-sans for good look.

Below is the screenshot of the page: 

image

Fixes : #17

### CODE REVIEW

**Comment by shuklamaneesh23** [comment]
Great work, @AMANVISHWAKARMA27! Everything looks good except for the font change. Please revert it to the previous version and let me know once it's done.

Thanks!

**Comment by AMANVISHWAKARMA27** [comment]
Sure @shuklamaneesh23 , will do asap

**Comment by AMANVISHWAKARMA27** [comment]
@shuklamaneesh23 , i have made the changes as per your requirements. Kindly review them.

**Comment by shuklamaneesh23** [comment]
Dear @AMANVISHWAKARMA27 ,the attachments that you have provided in the PR description are not visible and also make sure to remove that extra dotenv dependency as it is a separate thing.
Kindly go through the contributing.md before making such things.
Thanks!

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "axios": "^1.7.2",
     "cf_out_of_bounds": "file:",
     "cheerio": "^1.0.0",
+    "dotenv": "^16.4.5",
     "generative-ai": "^0.1.4",
```
Comment (shuklamaneesh23) [inline package.json:19]: remove this from dependency from line no 18.

File: src/app/page.js [hunk 2]
```diff
@@ -149,28 +143,28 @@ export default function Home() {
 
   return (
     <div className="min-h-screen flex flex-col items-center justify-center bg-gray-800 p-4">
-      <h1 className="text-2xl font-bold mb-4">Codeforces Submission Scraper</h1>
+      <h1 className="text-2xl font-bold mb-4 ">Codeforces Submission Scraper</h1>
       <form onSubmit={handleSubmit} className="w-full max-w-md">
```
Comment (shuklamaneesh23) [inline src/app/page.js:147]: the change in line number 152 is unnecessary.

**Comment by AMANVISHWAKARMA27** [comment]
@shuklamaneesh23 , you can review now.

**Comment by AMANVISHWAKARMA27** [comment]
> Dear @AMANVISHWAKARMA27 ,the attachments that you have provided in the PR description are not visible and also make sure to remove that extra dotenv dependency as it is a separate thing. Kindly go through the contributing.md before making such things. Thanks!

I have fixed the issue. Please have a look again.

File: src/app/page.js [hunk 3]
```diff
@@ -195,7 +189,7 @@ export default function Home() {
         </select>
         <button
           type="submit"
-          className="w-full bg-blue-500 text-white py-2 rounded-lg hover:bg-blue-600"
+          className="w-full h-[5vh] bg-blue-500 text-white py-2 rounded-lg hover:bg-blue-600 font-bold mb-4"
           disabled={loading}
         >
```
[the remark this answers is not in the document]
Comment (AMANVISHWAKARMA27) [reply src/app/page.js:194]: correcting it @shuklamaneesh23

[the remark this answers is not in the document]
Comment (AMANVISHWAKARMA27) [reply src/app/page.js:194]: @shuklamaneesh23 , I have made the changes. Please review.

**Comment by shuklamaneesh23** [comment]
Great work @AMANVISHWAKARMA27 ,closing this issue.
Thanks for the contribution.!

**Pull request merged**